### PR TITLE
Update/1.0.0

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update to 1.0.0
- reorder recipe

https://github.com/LLNL/zfp/releases/tag/1.0.0
https://github.com/LLNL/zfp/tree/1.0.0

### CODE REVIEW

**Review by skupr-anaconda — COMMENTED** [review]
LGTM.
Should we test `zfp` with a downstream package imagecodecs?

**Comment by cbouss** [comment]
> LGTM. Should we test `zfp` with a downstream package imagecodecs?

imagecodecs pins to: `zfp >=0.5.5,<1.0a0`, so this version won't be used.
This build is more to get a working zfpy on 3.11.

**Review by skupr-anaconda — APPROVED** [review]

**Review by ELundby45 — APPROVED** [review]

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -80,20 +65,23 @@ outputs:
         - ninja    # [win]
       host:
         - python
-        - numpy
-        - cython
+        - numpy {{ numpy }}
+        - cython 0.29.33
         - {{ pin_subpackage('zfp', exact=True) }}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:70]: I guess setuptools and wheel are missing in host

Comment (ELundby45) [reply recipe/meta.yaml:70]: Since we are not calling a `setup.py`, running `pip install` or generating wheel files, I don't think those are needed.

**Review by skupr-anaconda — APPROVED** [review]

**Comment by ELundby45** [comment]
@cbouss Could you review once you are back? I couldn't request it through the UI since this was originally your PR.

**Review by varlackc — APPROVED** [review]
LGTM

**Comment by cbouss** [comment]
That looked good to me! Thank you for having tackled this!

**Pull request merged**